libs/ts/contracts/scripts/test.ts

```diff
@@ -0,0 +1,28 @@
+import { ethers } from 'hardhat';
```

Comment:
This script looks like a debugging helper that was perhaps useful to you, but I don't think it's general to be useful for everybody.